packages/typography/src/Typography.types.ts

```diff
@@ -0,0 +1,30 @@
+/* eslint-disable @typescript-eslint/indent */
+import React from 'react';
+
+export interface ComponentProps {
+  /* `variant` - typography variant */
+  variant?:
+    | 'display'
+    | 'heading'
+    | 'subHeading'
+    | 'body'
+    | 'caption'
+    | 'overline'
+    | 'button';
+  /* `size` - variant size */
+  size?: 1 | 2 | 3 | 4 | 5 | 6;
```

Comment:
Is there a planned naming-conventions or nomenclature for the variant size instead of magic numbers. Especially that one can think is 1->5 ascending or descending order in terms of size?